azure-communication-ui/azure-communication-ui/src/main/res/values/azure_communication_ui_feature_flags.xml

```diff
@@ -11,9 +11,9 @@
     <bool name="azure_communication_ui_feature_flag_bluetooth_audio">false</bool>
 
 
-    <!-- For Tests -->
+    <!-- For Tests & Defaults -->
     <string name="azure_communication_ui_feature_flag_test_label">Test</string>
-    <bool name="azure_communication_ui_feature_flag_test_false">false</bool>
-    <bool name="azure_communication_ui_feature_flag_test_true">true</bool>
+    <bool name="azure_communication_ui_feature_flag_default_false">false</bool>
```

Comment:
this is a bit odd to me
normally, i would expect
<bool name="azure_communication_ui_feature_flag_default">false</bool>
what is the reason behind using _false and _true

Reply:
Need resource id's for the default values. 

If most features will be false by default, it makes sense to have a default key we can just re-use (less chance of accidental toggling).

Originally needed resource id's for tests, hence the "test" prefix before.

Reply:
It's updated in the branch to have 1 flag per feature, the defaults are back to test_ prefix, and only used in text